api/app.py

```diff
@@ -97,6 +100,57 @@ def try_grant_points(source_user_email, mentioned_user_email, points):
     """
     return None
 
+# Create an event listener for users joining the #general channel
+@slack_events_adapter.on("member_joined_channel")
+def handle_user_joined_channel(event_data):
+    message = event_data["event"]
+
+    # Only create accounts when certain channels are joined.
+    if message["channel"] != TEST_CHANNEL and message["channel"] != GENERAL_CHANNEL:
+        print("member joined a channel we don't care about...")
+        return
+
+    # Get user info and add them to the DB.
+    joined=message["user"]
+    joined_user = slack_client.users_info(user=joined)
+    joined_user_email = joined_user["user"]["profile"]["email"]
+    team = try_add_user(joined_user_email)
+    if team is None:
+        msg = """Hey <@{joined}> :wave: I'm <@{snitch}>! Looks like you joined #general but something went wrong so I wasn't able to assign you a team. 
```

Comment:
This seems to imply that we do actually care about the `#general` channel, whereas above we return early when the channel is `#general`

Reply:
above we skip when we're *not* in general

Reply:
Good point, coolio :)